src/components/new-safe/OverviewWidget/index.tsx

```diff
@@ -26,12 +26,20 @@ const OverviewWidget = ({ safeName }: { safeName: string }): ReactElement | null
           <SafeLogo alt="Safe logo" width={LOGO_DIMENSIONS} height={LOGO_DIMENSIONS} />
           <Typography variant="h4">Your Safe preview</Typography>
         </div>
-        {rows?.map((row) => (
-          <div key={row.title} className={css.row}>
-            <Typography variant="body2">{row.title}</Typography>
-            {row.component}
+        {wallet ? (
+          rows.map((row) => (
+            <div key={row.title} className={css.row}>
+              <Typography variant="body2">{row.title}</Typography>
+              {row.component}
+            </div>
+          ))
+        ) : (
+          <div className={css.row}>
+            <Typography variant="body2" color="border.main" textAlign="center" width={1}>
+              Connect your wallet to continue
+            </Typography>
           </div>
-        ))}
+        )}
```

Comment:
Let's move this ternary to `rows[0]`. There is already the `wallet` check there.

Reply:
It would be nice but after trying, it will still display the other values if they exist which makes the message "Connect your wallet to continue" a bit confusing.
<img width="314" alt="Screenshot 2022-11-21 at 15 20 06" src="https://user-images.githubusercontent.com/5880855/203077972-3a6f108b-f2bb-4754-8c4d-95dbd823c3d6.png">